### PR TITLE
update of scrollbar for pages

### LINKED ISSUE #279: Enhance Scrollbar Design for Improved User Experience
### Description:
- The scrollbar is an essential UI element that greatly influences the user experience while navigating through content-rich applications and websites. 
- my aims is to enhance the scrollbar design to improve usability, aesthetics, and user engagement across our platform.

### PR DESCRIPTION
**Pull request by Lobi29**
<h1 align="center">Fixes #279 </h1>

Untitled drawing (6)

### Added Feature

- Enhance the scrollbar functionality by dynamically adapting its thumb color to the current color theme
- hiding the scrollbar on devices smaller than 767px, fostering a mobile-centric experience that prioritizes seamless navigation and user interaction

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -130,3 +130,31 @@ html.dark {
...
+    background-color: var(--primary-500);
+    border-radius: 14px;
+    border: 3px solid var(--background);
+  }
+}
+
+.home-campaign-glowing-icon-glow-2 {
+  background-color: var(--primary-700);
+  filter: blur(20px);
+}
```
Comment (sainak) [inline app/globals.css:160]: Missing support for modern browsers
https://developer.chrome.com/docs/css-ui/scrollbar-styling#supporting_older_browser_versions

**Review by sainak — CHANGES_REQUESTED** [review]

File: app/globals.css (hunk 1 above)
Comment (Lobi29) [reply app/globals.css:160]: Got it 💯,

Comment (rithviknishad) [inline app/globals.css:160]: Why is this required? It's not used anywhere if I'm correct.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by Lobi29** [comment]
Hey, @rithviknishad  @sainak 

Regarding the scrollbar functionality, instead of relying on traditional CSS methods, I propose leveraging the "tailwind-scrollbar" package. It boasts a weekly download count exceeding 100k+ across both modern and legacy browsers, making it a robust solution for our needs. Given the frequency with which we'll be implementing scrollbars across various components, using this package seems both practical and efficient.

I'm prepared to integrate this package into our workflow if you believe it aligns with our objectives. Otherwise, I'm fully capable of proceeding with the conventional methods outlined in the documentation.🚀

File: app/globals.css (hunk 1 above)
Comment (Lobi29) [reply app/globals.css:160]: Yes, it's a mistake

**Comment by Lobi29** [comment]
Hey @sainak , I have added the features of tailwind-scrollbar package in our website 👍

File: tailwind.config.js [hunk 2]
```diff
@@ -63,5 +63,8 @@ module.exports = {
       },
     },
   },
-  plugins: [require("@tailwindcss/typography")],
+  plugins: [
+    require("tailwind-scrollbar")({ nocompatible: true }),
```
Comment (sainak) [inline tailwind.config.js:67]: `({ nocompatible: true })` defeated the purpose of the plugin

Comment (Lobi29) [reply tailwind.config.js:67]: working on it 🚀

Comment (Lobi29) [reply tailwind.config.js:67]: - Regarding why I use {nocompatible: true}

Screenshot from 2024-03-04 10-53-31

At present, this package lacks any impact on Firefox; however, it will seamlessly integrate with Firefox upon its next release. Additionally, Firefox offers limited options for customizing the scrollbar interface.

**Comment by rithviknishad** [comment]
@Lobi29 closing as stale + conflicts + not a requirement. Let's keep the design minimal and consistent across all browsers.

Feel free to re-open once if we can do this for all browsers consistently.
